_posts/2020-01-28-Tips-for-creating-open-source-projects-that-are-contributor-friendly.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Tips For Creating Contributor Friendly Open Source Projects
+date: 2020-01-28 06:31:01
+categories: [open-source]
+author: bronzdoc
+
+---
+
+At Ombu Labs we are firm believers of open source. We use open source but also like to contribute back to the community and we have open sourced a few projects.
+
+We know that contributing to open source can be a difficult task so we have been looking for easier ways help to onboard new and past contributors.
+Here is a list of tips and tools that we believe that can help you make your open source project a more friendly space.
+
```

Comment:
@bronzdoc you missed the `more` tag. It shows the entire article in the index otherwise

```suggestion

<!--more-->

```